RDSExpoApp/app/(tabs)/_layout.tsx

```
@@ -0,0 +1,35 @@
+  return (
+    <Tabs
+      screenOptions={{
+        tabBarActiveTintColor: Colors[colorScheme].tint,
+        headerShown: false, 
+      }}>
+      <Tabs.Screen
+        name="index"
+        options={{
+          title: 'index',
+          tabBarIcon: ({ color }) => (
+            <FontAwesome name="code" size={28} color={color} style={{ marginBottom: -3 }} />
```

ESLint

[error] Replace `·name="code"·size={28}·color={color}·style={{·marginBottom:·-3·}}` with `⏎··············name="code"⏎··············size={28}⏎··············color={color}⏎··············style={{·marginBottom:·-3·}}⏎···········`

(prettier/prettier)